loop.rb

```diff
@@ -8,6 +9,7 @@
     puts
 end
 puts `fn routes update myapp /fdk-ruby --format json`
+sleep 5
```

Comment:
Why the sleep?

Reply:
Because if you don't have the sleep, then the first couple of calls go wrong.

For example, I created a fresh app with `fn init runtime=ruby`.

Running the loop without sleep gives:

```
myapp /ruby updated
Hot
{"message":"Hello name!"}
time: 0.740203037

{"message":"Hello name!"}
time: 0.733902358

{"message":"Hello dawg 2!"}
time: 0.535276166

{"message":"Hello dawg 3!"}
time: 0.120919152

{"message":"Hello dawg 4!"}
time: 0.111981959
```

If I put the sleep back in I get:

```
myapp /ruby updated
Hot
{"message":"Hello dawg 0!"}
time: 0.590667239

{"message":"Hello dawg 1!"}
time: 0.11935071

{"message":"Hello dawg 2!"}
time: 0.132559136

{"message":"Hello dawg 3!"}
time: 0.161297556

{"message":"Hello dawg 4!"}
time: 0.138033515
```

(as expected).

I'll investigate why this is happening, but the sleep is a workaround for now.